### PR TITLE
[CORE] Place .jshintrc to root folder

### PR DESCRIPTION
**Pull request by chunghha**
This is simply not to display ES6 syntax warnings.

Signed-off-by: Chris Ha <chunghha@users.noreply.github.com>

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -328,10 +328,11 @@ module.exports = {
         "vendor"
       ],
       "minChunks": (module) => {
-                return module.resource
```
Comment (hippee-lee) [inline webpack.config.js]: It looks like these are formatting changes not related to the file above. Can you please revert them?

Comment (chunghha) [reply webpack.config.js]: Without this change, we see these warnings when jshint runs. That's why I changed. Do you still want to revert?

webpack.config.js: line 332, col 21, Misleading line break before '&&'; readers may interpret this as an expression boundary.
webpack.config.js: line 333, col 25, Misleading line break before '||'; readers may interpret this as an expression boundary.
webpack.config.js: line 334, col 25, Misleading line break before '||'; readers may interpret this as an expression boundary.

Comment (hippee-lee) [reply webpack.config.js]: I see, yeah this looks good since jshint is also deprecating the `laxbreak` and `laxcomma` settings.

**Comment by hippee-lee** [comment]
Thanks for submitting the PR. We discussed adding `jshint` to the project and decided not to include it at this time.

**Pull request closed without merging**